<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="ADMIN-PC" name="qa.impListeners.testcases.TC002_CreateLead2" tests="2" failures="0" timestamp="21 Apr 2020 04:40:09 GMT" time="107.503" errors="1">
  <testcase name="createNewLead2" time="55.471" classname="qa.impListeners.testcases.TC002_CreateLead2">
    <error type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException
at qa.impListeners.utils.Reporter.reportStep(Reporter.java:113)
at qa.impListeners.base.SeleniumBase.locateElement(SeleniumBase.java:96)
at qa.impListeners.pages.HomePage.clickCRMSFA(HomePage.java:24)
at qa.impListeners.testcases.TC002_CreateLead2.createNewLead2(TC002_CreateLead2.java:33)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:483)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.retryFailed(Invoker.java:838)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1009)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:744)
]]>
    </error>
  </testcase> <!-- createNewLead2 -->
  <testcase name="createNewLead2" time="52.032" classname="qa.impListeners.testcases.TC002_CreateLead2">
    <skipped/>
  </testcase> <!-- createNewLead2 -->
</testsuite> <!-- qa.impListeners.testcases.TC002_CreateLead2 -->
